WhaleSpotting/react-app/src/components/Button.tsx

```diff
@@ -31,13 +32,16 @@ export function Button({ style, text, onClick, dataTestId, minWidth25 = false, l
     case Style.reject:
         styleClass = "reject-button";
         break;
+    case Style.hidden:
```

Comment:
instead of adding hidden as a style, can we add it as a prop thats passed in with default value false. Then we can use the hidden attribute on both the link section and the button section

Reply:
Addressed.

Reply:
So instead of doing this, we want to use the attribute hidden, will add a suggestion with that